test/integration/recipe.test.js

```diff
@@ -74,7 +90,23 @@ describe('Recipes', function () {
       expect(response.status).to.equal(400)
     })
 
-    it('invalid attachment id errors', async function () {
+    it('should accept valid body', async function () {
+      const newRecipe = {
+        externalId: 'foobar3000',
+        name: 'foobar3000',
+        imageAttachmentId: '00000000-0000-1000-8000-000000000000',
```

Comment:
how does this work with the casing changes

Reply:
because it expects payload to be in camelCase but before updating to the db it maps